components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+      fn convert_i64_to_string(path: &str, json: &str) -> String;
+      fn convert_string_to_i64(path: &str, json: &str) -> String;
+    }
+}
+
+// Converts int64/uint64 value to string and serializes json with new values.
```

Comment:
what happens if it's outside the range for the type?

Reply:
I see below that it returns null which I guess translates to an empty string in cpp? @evq @rillian @antonok-edm is there an equivalent of optional here or do we have a pattern to use for this?

Reply:
Rust has `Option<T>` but that's not yet supported by `cxx`.